src/input-processing/inputProcessing.js

```diff
@@ -308,4 +341,4 @@ function removeLinks(input) {
     input = replaceSpecialLetters(input);
     input = input.replace(LINK_STR, EMPTY_STR);
     return input;
-}
+}
```

Comment:
"No new line at the end of the file" It breaks the POSIX standards. You can read more about this [here](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline/729725#729725).